components/fonts/font.rs

```diff
@@ -191,18 +197,30 @@ pub struct FontDescriptor {
     pub style: FontStyle,
     pub variant: font_variant_caps::T,
     pub pt_size: Au,
+    #[ignore_malloc_size_of = "can't measure font variations"]
```

Comment:
We can! Add an entry like https://github.com/servo/servo/blob/7fc2f3130968aad558238d35419b9541730c2911/components/malloc_size_of/lib.rs#L801 and you can remove this line.

Reply:
Thanks!